easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'PheWeb'
+version = '1.1.20'
+versionsuffix = '-Python-%(pyver)s'
+
+
+homepage = 'https://github.com/statgen/pheweb'
+description = """A tool for building PheWAS websites from association files"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Pysam', '0.15.3'),
+    ('openpyxl', '3.0.3', versionsuffix, ('GCCcore', '8.3.0')),
+    ('Flask', '1.1.2', versionsuffix, ('GCCcore', '8.3.0')),
```

Comment:
is specifying the subtoolchain necessary?

Reply:
@migueldiascosta certainly open to better suggestions here - I used the first formulation I found that worked.

Reply:
```
    ('openpyxl', '3.0.3', versionsuffix),
    ('Flask', '1.1.2', versionsuffix),
```

works for me... but maybe it depends on some easybuild toolchain-related configuration setting you are using?

Reply:
done in 6d6131f